elements/pl-file-editor/pl-file-editor.js

```diff
@@ -46,6 +46,12 @@ window.PLFileEditor = function (uuid, options) {
     this.editor.setFontSize(12);
   }
 
+  if (localStorage.getItem('pl-file-editor-keyboardHandler') !== 'ace/keyboard/default') {
```

Comment:
Does Ace complain if you pass `ace/keyboard/default` to `setKeyboardHandler`? If not, what is the if/else here and elsewhere accomplishing?

Reply:
I tested it out before. Despite we have a default like keybindings, it is not the default in the ace editor. This is the reason that I add all the if statements to manually pass in the null value to the setKeyHandler() method.

Reply:
Ah, gotcha. I believe the suggestion from @louisunlimited is a nice way to improve this!

Reply:
Understood, I will work on it tonight.